plugins/scaffolder/src/components/fields/EntityNamePicker/validation.ts

```diff
@@ -17,6 +17,9 @@
 import { FieldValidation } from '@rjsf/utils';
 import { KubernetesValidatorFunctions } from '@backstage/catalog-model';
 
+/**
+ * @public
```

Comment:
I don't think that these need to be public right? Same as the schemas?

Reply:
Well, the existing ones were so followed the pattern. Maybe some clean up necessary here as well?

Reply:
Cleaned them all off; not completely sure if there was a reason they were public in the first place?